tests/integration_tests/performance/test_boottime.py

```diff
@@ -60,6 +60,11 @@ def test_no_boottime(test_microvm_with_api):
     global_props.host_linux_version == "6.1",
     reason="perf regression under investigation",
 )
+@pytest.mark.skipif(
```

Comment:
PR title and commit message still say Cascade Lake

Reply:
Fixed.

Reply:
nit: officially it is Skylake. Though later they switched to separating with Cascade Lake etc

https://en.wikipedia.org/wiki/Skylake_(microarchitecture)